internal/resources/schemas/nsx_t_network_schemas.go

```diff
@@ -0,0 +1,149 @@
+package schemas
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const (
+	DSRouter = "hpegl_vmaas_router"
+)
+
+func StaticNetworkSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeList,
+		Optional:    true,
+		Description: "static Network configuration",
+		MaxItems:    1,
+		ExactlyOneOf: []string{
+			"static_network",
+			"dhcp_network",
+		},
+		ConflictsWith: []string{
+			"dhcp_network",
+		},
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"code": {
+					Type:        schema.TypeString,
+					Computed:    true,
+					Description: "Network Type code",
```

Comment:
why code, ids are here ? are they specific to static network ?

Reply:
no updated